### DISCUSSION TITLE
Welcome to stando Discussions!

### DISCUSSION THREAD

**Discussion by maxwowo** [maxwowo/stando#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raredevice** [comment]
Hi there. Lovely little app and really useful. However, since updating to M3 Mac and Sequoia every time I start up it opens Stando's preferences window. Am I doing something wrong, or does it need a tweak for the new system?

**Reply by maxwowo** [reply]
Hmm that's strange 🤔 I'll look into it. Thanks for reporting it!

**Reply by maxwowo** [reply]
Fixed! Turned out to be a new SwiftUI behavior that got introduced in Sequoia 15. I've made the necessary changes and submitted the new build for distribution. Expect a new update in a day or two's time 😄

**Reply by raredevice** [reply]
Wow! I'm so glad I flagged it. What a great start to the day!! Thanks so much. If you're interested in a new feature idea, I've always thought the option of having it start at a specific time would be nice. For example, I switch on my Mac before I take the kids to school, but I want the timer to begin 9am, ready for the work day.

Have a great day and thank you.

**Reply by maxwowo** [reply]
No worries!

Interesting idea, I feel like it's a bit niche for a feature though. I currently intend to keep Stando's feature set as minimal as possible so that it doesn't get bogged down trying to do too many things at the same time. Thanks for the suggestion anyway though 😄

**Reply by maxwowo** [reply]
New version just landed in the App Store 👍

**Reply by raredevice** [reply]
Amazing work. Thanks so much. I've popped a little review up to say thanks.
